Add suspend help page.

mwm opened the pull request:
As requested. Probably needs a bit of cleanup.

faho commented:
Do we want a bit more info on what "suspend" means in this context? And that it does those checks to avoid a dangling term? Also, is "subshell" applicable here? In bash-land, that's pretty much about the "($COMMANDS)" construct, I've not heard it used for a `bash` executed in another one.

mwm commented:
Updated. The checks also prevent suspending a shell that isn't in a windowing environment, say via an SSH or on the console before starting x or in single-user mode. But I added more text about that as well.

faho commented:
Did you forget to push or commit?

mwm commented:
Whoops. Yup. There now.

faho commented:
The text looks good now, I'll try to check the formatting tomorrow.

faho commented:
Thanks btw for going the extra mile - docs are frequently ignored, and plenty of people just want to drop off a particular patch and leave.

faho commented:
Merged as 32a3e15309a0f4296db49261163c575a6468b2c8. Thanks.
